README.md

```diff
@@ -4,201 +4,82 @@ The Observability Operator (previously known as Monitoring Stack Operator) is a
 Kubernetes operator which enables the management of Monitoring, Logging and
 Tracing stacks through Kubernetes CRDs.
 
-The project is based on the [controller-runtime](https://github.com/kubernetes-sigs/controller-runtime) library.
+The project relies heavily on the [controller-runtime](https://github.com/kubernetes-sigs/controller-runtime) library.
 
 ## Trying out the Operator
 
-### OLM
+Observability Operator requires Operator Lifecycle Manager (OLM) to be running
+in cluster. The easiest way is to use OpenShift where OLM is preinstalled.
```

Comment:
Hm that is not really accurate is it? Or is there an intention to require OLM?

Reply:
Yes, we require OLM since the certs for admission-webhook are created by OLM now.